src/pages/patientView/timeline/timeline_helpers.tsx

```diff
@@ -24,6 +24,7 @@ import { ISampleMetaDeta } from 'pages/patientView/timeline/TimelineWrapper';
 import { ClinicalEvent } from 'cbioportal-ts-api-client';
 import { getColor } from 'cbioportal-frontend-commons';
 import ReactMarkdown from 'react-markdown';
+import { LegendItem } from 'cbioportal-clinical-timeline/src';
```

Comment:
This was also caught by `ci/circleci: check_incorrect_import_statements`

```suggestion
import { LegendItem } from 'cbioportal-clinical-timeline';
```